### PR TITLE
fix: DIA-1740: 'int' not iterable when Rating GT empty

### PR DESCRIPTION
**Pull request by hakan458**
Most control tags' labels are stored in a list, for some like Number and Rating it is just a single int. We were not handling the case where the annotation to compare against is emtpy, for a Rating / Number tag

Fixed DIA-1741 here as well. We were never calculating pairwise comparisons correctly. We were catching an exception in the case both annotation and pred have non-empty pairwise selection, but issue manifested when  GT was empty
We were using the wrong key all along

### CODE REVIEW

File: evalme/classification.py [hunk 1]
```diff
@@ -96,7 +98,7 @@ class ChoicesEvalItem(ClassificationEvalItem):
 
 
 class PairwiseEvalItem(ClassificationEvalItem):
-    SHAPE_KEY = 'pairwise'
```
Comment (pakelley) [inline evalme/classification.py:99]: why is this change necessary?

File: evalme/classification.py [hunk 2]
```diff
@@ -96,7 +98,7 @@ class ChoicesEvalItem(ClassificationEvalItem):
...
+    SHAPE_KEY = 'selected'
```
Comment (pakelley) [inline evalme/classification.py:101]: this seems pretty strange, are we pretty confident that this should definitely be the value here? And that there isn't something else depending on it being "pairwise"? (maybe worth leaving a comment about why it's "selected" instead of "pairwise"?)

Comment (hakan458) [reply evalme/classification.py:101]: Whats strange is that it should have been 'selected' this entire time and it was not caught because we have a try/except that was silencing the error. However when comparing an empty annotation to a GT the error occurs, and we dont make such comparisons in normal LS it seems. But I will make sure this doesnt affect anything else

Comment (hakan458) [reply evalme/classification.py:101]: This was added 5 years ago and perhaps the key that we used in annotations back then was 'pairwise' but it most definitely is not now. It is 'selected' 
This tells me that no one has noticed that metrics/scores are wrong for pairwise tag comparisons lol 

https://github.com/HumanSignal/label-studio-evalme/commit/f4408434af8e736fa1585234a53ad70a97ba39f1

Comment (hakan458) [reply evalme/classification.py:101]: image

**Review by matt-bernstein — APPROVED** [review]
